src/libraries/System.Linq.Queryable/tests/ZipTests.cs

```diff
@@ -87,8 +87,8 @@ public void TupleNames()
             int[] first = new int[] { 1 };
             int[] second = new int[] { 2 };
             var tuple = first.AsQueryable().Zip(second.AsQueryable()).First();
-            Assert.Equal(tuple.Item1, tuple.First);
-            Assert.Equal(tuple.Item2, tuple.Second);
+            Assert.Equal(tuple.First, tuple.First);
```

Comment:
These changes are unrelated. Could you please remove them?

Reply:
With analyzer update, [SA1142](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1142.md) was violating at the usage of Item1/Item2 rather than the actual names used in signature.